Apps/phipqualityscore/ipqualityscore_consts.py

```diff
@@ -0,0 +1,44 @@
+# File: ipqualityscore_consts.py
+#
+# Licensed under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.txt)
+
+IPQUALITYSCORE_DOMAIN = 'https://ipqualityscore.com'
+
+IPQUALITYSCORE_API_TEST = 'https://ipqualityscore.com/api/json/ip/{apikey}/8.8.8.8'
+IPQUALITYSCORE_API_URL_CHECKER = 'https://ipqualityscore.com/api/json/url/{apikey}/{url}'
+IPQUALITYSCORE_API_IP_REPUTATION = 'https://ipqualityscore.com/api/json/ip/{apikey}/{ip}'
+IPQUALITYSCORE_API_EMAIL_VALIDATION = 'https://ipqualityscore.com/api/json/email/{apikey}/{email}'
+
+IPQUALITYSCORE_APP_KEY = 'app_key'
+IPQUALITYSCORE_MSG_QUERY_URL = 'Querying URL: {query_url}'
+IPQUALITYSCORE_MSG_CONNECTING = 'Polling IPQualityScore site ...'
+IPQUALITYSCORE_SERVICE_SUCC_MSG = 'IPQualityScore Service successfully executed.'
+IPQUALITYSCORE_SUCC_CONNECTIVITY_TEST = 'Test Connectivity passed'
+IPQUALITYSCORE_ERR_CONNECTIVITY_TEST = 'Test Connectivity failed'
+IPQUALITYSCORE_MSG_GOT_RESP = 'Got response from IPQualityScore'
+IPQUALITYSCORE_NO_RESPONSE = 'Server did not return a response \
+                         for the object queried'
+IPQUALITYSCORE_SERVER_CONNECTION_ERROR = 'Server connection error'
```

Comment:
let's be consistent about using either `ERROR` or `ERR`

Reply:
Fixed